web/templates/message_avatar.hbs

```diff
@@ -1,6 +1,9 @@
 <div class="u-{{msg/sender_id}} message-avatar sender_info_hover view_user_card_tooltip" aria-hidden="true" data-tooltip-template-id="view-user-card-tooltip-template">
-    <div class="inline_profile_picture {{#sender_is_guest}} guest-avatar{{/sender_is_guest}}">
-        <img src="{{small_avatar_url}}" alt="" class="no-drag"/>
+    <div class="inline_profile_picture
+      {{#if sender_is_bot}}
+      {{#if sender_has_guest_role}} bot-avatar guest-avatar
+      {{else}} bot-avatar{{/if}}{{else if sender_is_guest}} guest-avatar
```

Comment:
This conditions are somewhat hard to read. Wouldn't it be better to just have it like this `{{#if sender_is_bot}}bot-avatar{{else if sender_is_guest}}guest-avatar{{/if}}`? It will keep the behavior as we want for guest bots as well. There is no need to have both the classes for guest bots since these classes are only used to show the triangle indicator in profile picture and we always show one of them only.

And this similar change should be done at all the places.